transports/sqs/index.md

```diff
@@ -80,14 +80,40 @@ NOTE: NServiceBus will perform [immediate](/nservicebus/recoverability/#immediat
 
 ### AmazonSQSException: Request is throttled
 
-SQS can throttle some send operations. When that happens the following exception is logged.
+Amazon SQS could apply throttling, it can deal with a very large continious throughput but if there are sudden spikes the service can apply throttling.
+
+When throttling happens the following exception is logged:
+
 ```
 2017-11-14 23:10:24,314|ERROR|18|NServiceBus.Transports.SQS.MessageDispatcher|Exception from Send.
 Amazon.SQS.AmazonSQSException: Request is throttled. ---> Amazon.Runtime.Internal.HttpErrorResponseException: The remote server returned an error: (403) Forbidden. ---> System.Net.WebException: The remote server returned an error: (403) Forbidden.
 ```
 
-For incoming messages this can be safely ignored as [recoverability feature](/nservicebus/recoverability/) should resolve it. For outgoing messages sent outside of a handler context, send operation can fail. Sending messages in bulk increases the chance of being throttled on send.
+Throttling is more likely to happen when sending a large number messages concurrently (in parallel). For example when using async/await, using a list of tasks to be executed with `Task.WaitAll()` could cause throttling and partial message delivery if any of the sends result in an thottling error.
+
+To avoid possible Amazon throttling errors, limit the maximum number of concurrent sends. For example, allow only a small amount of messages to be sent concurrently as outlined in the [sending large amount of messages](/nservicebus/handlers/async-handlers.md#concurrency-large-amount-of-concurrent-message-operations) guidelines or don't use concurrent sends at all and sequentially send your messages.
+
+Throttling can happen during any send or receive operation and can happen during the following scenarios:
+
+- Incoming message (receiving)
+- Sending from within a handler
+- Sending outside of a handler
+
+
+#### Incoming message (receiving)
+
+For incoming messages throttling errors can be safely ignored as the message pump will just try to fetch the next available message again.
+
+#### Sending from within a handler
+
+The [recoverability feature](/nservicebus/recoverability/) should resolve it as any message sends and fails results in an exception which will be processed by the recoverability logic. If an incoming message would continiously result in a throttling error then the message will eventually be moved to the error queue.
+
+A throttling error could result in ghost messages. Ghost messages are messages send even though the handler failed due to not being able to transmit all messages to the transport succesfully. This makes a throttling error similar to any other error could and can occur both with the default [batched message dispatch](/nservicebus/messaging/batched-dispatch) or when using [immediate dispatch](nservicebus/messaging/send-a-message#dispatching-a-message-immediately).
```

Comment:
Ghost messages are only relevant if a handler produces multiple messages

Reply:
The second sentence is quite complex and hard to follow.

You want to say that ghost messages can occur regardless of the dispatch mode?

Reply:
> Ghost messages are only relevant if a handler produces multiple messages

@danielmarbach I think that isn't true, also if it only sends a single message and the incoming message isn't ACKd or committed succesfully.

Reply:
> You want to say that ghost messages can occur regardless of the dispatch mode?

Yes, we do not have a definition of what a ghost message is and googling on the term also does not give you a definition.

Reply:
@ramonsmits yeah you are right. But that is by definition the case with transports that only support TX level receive only right? So should we just link to that doco page instead?